examples/js/loaders/LDrawLoader.js

```diff
@@ -337,18 +599,161 @@ THREE.LDrawLoader = ( function () {
 
 				function finalizeObject() {
 
-					if ( ! scope.separateObjects && ! parentParseScope.isFromParse ) {
+					if ( scope.smoothNormals && parseScope.type === 'Part' ) {
+
+						smoothNormals( parseScope.triangles, parseScope.lineSegments );
+
+					}
+
+					var isRoot = ! parentParseScope.isFromParse;
+					if ( scope.separateObjects && ! isPrimitiveType( parseScope.type ) || isRoot ) {
+
+
+						const objGroup = parseScope.groupObject;
+						if ( parseScope.triangles.length > 0 ) {
+
+							objGroup.add( createObject( parseScope.triangles, 3 ) );
+
+						}
+
+						if ( parseScope.lineSegments.length > 0 ) {
+
+							objGroup.add( createObject( parseScope.lineSegments, 2 ) );
+
+						}
+
+						if ( parseScope.conditionalSegments.length > 0 ) {
+
+							var conditionalSegments = parseScope.conditionalSegments;
+							var lines = createObject( conditionalSegments, 2 );
+							lines.isConditionalLine = true;
+							lines.visible = false;
```

Comment:
I would leave the optional lines visible by default as you mentioned in the previous PR.

Reply:
Oh oops! Thanks I'll do that -- also once the other PRs are merged this one will be easier to take a look at. I know there's a lot of noise right now!